Makefile

```diff
@@ -10,7 +10,7 @@ TEST_TIMEOUT ?= 5s
 RELEASE_UID ?= $(shell id -u)
 RELEASE_GID ?= $(shell id -g)
 
-GOLANGCILINT_WANT_VERSION = 1.46.0
```

Comment:
should we bump this as well ?

https://github.com/cilium/cilium/blob/6a8fe290930f73da0311813259f93ac244b7004a/.github/workflows/lint-go.yaml#L58

Reply:
Good point, will do as part of https://github.com/cilium/cilium/pull/20061

Reply:
oops, did I just review PR with wrong repo in mind :rofl: :face_exhaling:

Reply:
:D fixed it up in this repo as well and repushed